transip/service/dns.py

```diff
@@ -0,0 +1,50 @@
+"""
+DNS related objects
+"""
+
+from suds.sudsobject import Object as SudsObject
+
+
+class DnsEntry(SudsObject):
+    """
+    Representation of the DnsEntry object
```

Comment:
Self-implied comment... This is _per definition_ the representation of the `DnsEntry` class.

Proposed:

    Representation of a DNS entry which could also be found in the control-panel for TransIP 
    and/or in DNS zone-files, etc..